qgis4/ansible/Vagrantfile

```diff
@@ -0,0 +1,20 @@
+Vagrant.configure(2) do |config|
+
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 4096
+    v.cpus = 4
+  end
+
+  config.vm.box = "debian/stretch64"
+
+  config.vm.define "qgis4" do |machine|
+    machine.vm.hostname = "qgis4"
+    machine.vm.network "private_network", ip: "192.168.77.2"
```

Comment:
Any reason for using a specific IP address? I usually let Vagrant/VirtualBox assign an IP for me...

Reply:
It's worth a try

Reply:
FYI it seems working without this `machine.vm.network`